static/app/components/statusIndicator.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+
+import Tooltip from 'sentry/components/tooltip';
+
+type Props = {
+  status: string;
+  tooltipTitle: string;
+};
+
+/**
+ * A badge/indicator at the beginning of the row that displays
+ * the color of the status level (Warning, Error, Success, etc)
+ *
+ */
+function StatusIndicator({status, tooltipTitle}: Props) {
+  let color: string = 'error';
+
+  if (status === 'muted') {
+    color = 'muted';
+  } else if (status === 'info') {
+    color = 'info';
+  } else if (status === 'warning') {
+    color = 'warning';
+  } else if (status === 'success' || status === 'resolved') {
+    color = 'success';
+  }
+
+  return (
+    <StatusLevel color={color}>
+      <BadgeTooltip title={tooltipTitle} />
+    </StatusLevel>
+  );
+}
+
+type BadgeTooltipProps = {
+  title?: string;
+};
+
+function BadgeTooltip({title}: BadgeTooltipProps) {
```

Comment:
i think because `StatusLevel` applies styles to the span below it, we should keep this all inline and not its own function